package/network/services/hostapd/files/hostapd.sh

```diff
@@ -765,6 +765,7 @@ wpa_supplicant_set_fixed_freq() {
 		NOHT) append network_data "disable_ht=1" "$N$T";;
 		HT20|VHT20) append network_data "disable_ht40=1" "$N$T";;
 		HT40*|VHT40*|VHT80*|VHT160*) append network_data "ht40=1" "$N$T";;
+		NOHT*|HT*) append network_data "disable_vht=1" "$N$T";;
```

Comment:
Note that this is a case, so NOHT, HT20 and HT40* won't fall into this option ...

Reply:
You could actually merge this case with the one following it.